### PR TITLE
Add public link to public api search get call

### PR DESCRIPTION
**Pull request by HonTastic2**
### Description of change

Added an extra line to also send the public link of an application if it has consultation

### CODE REVIEW

**Review by benbaumann95 — APPROVED** [review]

File: engines/bops_api/schemas/odp/v0.6.0/shared/definitions.json [hunk 1]
```diff
@@ -58,6 +58,10 @@
             "endDate": {
               "type": ["string", "null"],
               "format": "date"
+            },
+            "publicUrl": {
+              "type": ["string", "null"],
+              "format": "string"
```
Comment (EGiataganas) [inline engines/bops_api/schemas/odp/v0.6.0/shared/definitions.json:64]: How about you’re moving this change to latest version 0.7.0?

Comment (benbaumann95) [reply engines/bops_api/schemas/odp/v0.6.0/shared/definitions.json:64]: Yeah good catch.

@HonTastic2 We've just recently updated the schema version so we'll need to rebase this PR and add this to the 0.7.0 file before you can merge

**Review by benbaumann95 — CHANGES_REQUESTED** [review]
https://github.com/unboxed/bops/pull/1888/files#r1682783565

File: engines/bops_api/schemas/odp/v0.6.0/shared/definitions.json (hunk 1 above)
Comment (HonTastic2) [reply engines/bops_api/schemas/odp/v0.6.0/shared/definitions.json:64]: done

**Review by benbaumann95 — APPROVED** [review]

**Pull request merged**